appveyor.yml

```diff
@@ -1,4 +1,4 @@
-version: 2018.1.{build}
+version: 2018.3.{build}
```

Comment:
The SDK release versions do not tie in to the ARL release versions

Reply:
fixed